server/sonar-server/src/test/java/org/sonar/server/component/index/ComponentIndexScoreTest.java

```diff
@@ -146,6 +146,13 @@ public void do_not_match_wrong_file_extension() {
   }
 
   @Test
+  public void rank_a_higher_then_b() {
```

Comment:
this test should be renamed so that we understand that ordering fallbacks on name field is score is the same between two results.